users/schicklr/index.md

```diff
@@ -5,8 +5,39 @@ author: Robert Schickler
 ---
 Robert Schickler
 ================================
-I like pizza.
+<img src="images/schickler.jpg">
```

Comment:
<img src="{{ site.url }}users/schicklr/images/schickler.jpg" height="X" width="Y">

where X and Y are appropriate picture dimensions